server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -796,7 +796,7 @@ public synchronized void verifyIndexMetadata(IndexMetadata metadata, IndexMetada
             IndicesFieldDataCache indicesFieldDataCache = new IndicesFieldDataCache(settings, new IndexFieldDataCache.Listener() {
             });
             closeables.add(indicesFieldDataCache);
-            IndicesQueryCache indicesQueryCache = new IndicesQueryCache(settings);
+            IndicesQueryCache indicesQueryCache = new IndicesQueryCache(Settings.EMPTY);
```

Comment:
Did you mean to make this change?

Reply:
Yes, to avoid many warning/deprecation logs. The cache here is just a place holder.